File: mobile/lib/domain/entities/bill.dart
class Bill {
  final String id;
  final String userId;
  final String title;
  final double amount;
  final DateTime dueDate;
  final String? repeatCycle;
  final String status;

  Bill({
    required this.id,
    required this.userId,
    required this.title,
    required this.amount,
    required this.dueDate,
    this.repeatCycle,
    required this.status,
  });
}

File: app/Models/Bill.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Concerns\HasUuids;
use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Bill extends Model
{
    use HasFactory, HasUuids;

    protected $fillable = [
        'user_id',
        'title',
        'amount',
        'due_date',
        'repeat_cycle',
        'status',
    ];

    public function user()
    {
        return $this->belongsTo(User::class);
    }
}
